python/chemiscope/structures/__init__.py

```diff
@@ -66,6 +82,10 @@ def _list_atom_properties(frames):
 
     if adapter == "ASE":
         return _ase_list_atom_properties(frames)
+    elif adapter == "stk":
+        # Do not check, because stk does not contain its own properties
+        # and a dictionary must be added.
+        pass
```

Comment:
```suggestion
        # stk does not contain properties inside the structure objects
        return []
```

Reply:
Resolved with the _stk_list_*_properties functions that return `[]`